docs/whats-new/dotnet-docs-2021-09-01.md

```diff
@@ -0,0 +1,177 @@
+- [IL2040: Could not find embedded resource specified in a substitution file](../core/deploying/trim-warnings/il2040.md)
+- [IL2041: 'DynamicallyAccessedMembersAttribute' is not allowed on methods](../core/deploying/trim-warnings/il2041.md)
+- [IL2042: Could not find a unique backing field to propagate the 'DynamicallyAccessedMembersAttribute' annotation on a property](../core/deploying/trim-warnings/il2042.md)
+- [IL2043: 'DynamicallyAccessedMembersAttribute' on property conflicts with the same attribute on its accessor method](../core/deploying/trim-warnings/il2043.md)
+- [IL2044: Could not find any type in a namespace specified in a descriptor file](../core/deploying/trim-warnings/il2044.md)
+- [IL2045: Custom attribute is referenced in code but the trimmer was instructed to remove all of its instances](../core/deploying/trim-warnings/il2045.md)
+- [IL2046: All interface implementations and method overrides must have annotations matching the interface or overriden virtual method 'RequiresUnreferencedCodeAttribute' annotations](../core/deploying/trim-warnings/il2046.md)
+- [IL2048: Internal trimmer attribute 'RemoveAttributeInstances' is being used on a member](../core/deploying/trim-warnings/il2048.md)
+- [IL2049: Unrecognized internal attribute](../core/deploying/trim-warnings/il2049.md)
+- [IL2050: Correctness of COM interop cannot be guaranteed](../core/deploying/trim-warnings/il2050.md)
+- [IL2051: Property element does not have required argument `name` in custom attribute annotations file](../core/deploying/trim-warnings/il2051.md)
+- [IL2052: Could not find property specified in custom attribute annotations file](../core/deploying/trim-warnings/il2052.md)
+- [IL2053: Invalid value used in property element in custom attribute annotations file](../core/deploying/trim-warnings/il2053.md)
+- [IL2054: Invalid argument value in custom attribute annotations file](../core/deploying/trim-warnings/il2054.md)
+- [IL2055: Call to 'System.Type.MakeGenericType' cannot be statically analyzed by the trimmer](../core/deploying/trim-warnings/il2055.md)
+- [IL2056: A 'System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembersAttribute' annotation on a property conflicts with the same attribute on its backing field](../core/deploying/trim-warnings/il2056.md)
+- [IL2057: Unrecognized value passed to the `typeName` parameter of 'System.Type.GetType(String)'](../core/deploying/trim-warnings/il2057.md)
+- [IL2058: Parameters passed to 'Assembly.CreateInstance' cannot be statically analyzed](../core/deploying/trim-warnings/il2058.md)
+- [IL2059: Unrecognized value passed to the `type` parameter of 'System.Runtime.CompilerServices.RuntimeHelpers.RunClassConstructor'](../core/deploying/trim-warnings/il2059.md)
+- [IL2060: Call to 'System.Reflection.MethodInfo.MakeGenericMethod' cannot be statically analyzed by the trimmer](../core/deploying/trim-warnings/il2060.md)
+
+### Updated articles
+
```

Comment:
I'm surprised https://github.com/dotnet/docs/commits/main/docs/core/project-sdk/msbuild-props.md doesn't show up in this list.

Reply:
@gewarren For now, can you make a suggestion to add it by hand?  I'll look into why it wasn't added.

Reply:
I'll add a suggestion.

What's the criteria for making this "updated article" list? Overall it just seems really short to me.